docs/source/notes/faq.rst

```diff
@@ -18,57 +26,70 @@ please remove the installation directory from your MATLAB path.
+Note: The submodule throwing an error might be different than
+``external/lusolMex64bit``, but the command should work with any submodule.
 
-**Answer**: ``DM_ reactions`` are commonly demand reactions.
 
-Submodules
-----------
+On Windows, MATLAB R2016b crashes with CPLEX 12.7.1. Why?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When running ``git submodule update``, the following error message
-appears:
+When you experience an unexpected crash of MATLAB ``R2016b`` when running:
 
 .. code::
 
-    No submodule mapping found in .gitmodules for path 'external/lusolMex64bit'
+    >> changeCobraSolver('ibm_cplex')
 
-**Solution**: remove the cached version of the respective submodule
+or
 
 .. code::
 
-    git rm --cached external/lusolMex64bit
+    >> initCobraToolbox
+
+after having installed ``CPLEX 12.7.1``, the solver might not be
+correctly installed (see `this
+issue <https://github.com/opencobra/cobratoolbox/issues/802>`__).
 
-**Note**: The submodule throwing an error might be different than
-``external/lusolMex64bit``, but the command should work with any
-submodule.
+In order to fix this issue, follow these steps:
+
+-  Uninstall all older versions of CPLEX (e.g., ``12.6.3``)
+-  Uninstall CPLEX ``12.7.1``
+-  Restart your computer
+-  Install CPLEX ``12.7.1``. You will be prompted to install
+   ``Microsoft Visual C++ 2013``
+-  Download `this software
+   package <https://www.microsoft.com/en-us/download/details.aspx?id=40784>`__
+   and install ``Microsoft Visual C++ 2013 (x64)``
+-  Finish the installation of CPLEX ``12.7.1``
+-  Restart your computer
+-  Start MATLAB and the above commands again
 
 Parallel programming
 --------------------
 
-When running cobra code in a parfor loop, solvers (and other global
-variables) are not properly set.
+When running code in a parfor loop, solvers (and other global variables) are not properly set.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-| **Answer**: This is an issue with global variables and the matlab
-  parallel computing toolbox. Global variables are not passed on to the
-  workers of a parallel pool.
-| To change cobra global settings for a parfor loop, it is necessary to
-  reinitialize the global variables on each worker. The toolbox offers 
-  two helper functions for this purpose, which also take care of pathes, 
-  ``getEnvironment()`` and ``restoreEnvironment()``, which can be used 
-  as in the below example.
+This is an issue with global variables and the matlab
+parallel computing toolbox. Global variables are not passed on to the
+workers of a parallel pool. To change cobra global settings for a parfor loop, it is necessary to
+reinitialize the global variables on each worker. The toolbox offers
+two helper functions for this purpose, which also take care of pathes,
+``getEnvironment()`` and ``restoreEnvironment()``, which can be used
+as in the below example.
 
 .. code::
```

Comment:
`MATLAB` code has to be within a `.. code-block:: matlab` instead of `.. code::`.